fitbenchmarking/parsing/fitbenchmark_parser.py

```diff
@@ -31,12 +31,21 @@
     import_success['sasview'] = (False, e)
 
 
+# By design the parsers only have the requirment for one public method and many
+# private methods
+# pylint: disable=too-few-public-methods, too-many-branches
```

Comment:
These are already disabled in the pylintrc?

Reply:
Done